siege_utilities/geo/boundary_providers.py

```
@@ -227,3 +228,141 @@ def resolve_boundary_provider(country: str = 'US', **kwargs: Any) -> BoundaryPro
+            self._client = RDHClient(
+                username=self._username,
+                password=self._password,
+                **({"cache_dir": self._cache_dir} if self._cache_dir else {}),
+            )
+        return self._client
+
+    @property
+    def provider_name(self) -> str:
+        return 'Redistricting Data Hub'
+
+    def get_boundary(self, level: str, identifier: Optional[str] = None, **kwargs: Any):
```

Ruff (0.15.10)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**kwargs`

(ANN401)